src/hub.html

```diff
@@ -453,6 +470,24 @@
                 </a-entity>
             </template>
 
+            <template id="linked-media">
+                <a-entity
+                    class="interactable"
+                    body-helper="type: dynamic; mass: 1; collisionFilterGroup: 1; collisionFilterMask: 15;"
+                    is-remote-hover-target
+                    tags="offersRemoteConstraint: true;"
```

Comment:
Are you allowed to create a physics constraint between the remote and this object? I thought it was going to have a component attached that set its transform to followed your camera / avatar head around.

Reply:
i'm not familiar with this tag - you should only need to use the cursor with these objects. can this be removed?

Reply:
Yes, I think you should be able to remove this tag. The `offersRemoteConstraint` tag is meant to signal to the `constraints-system` that it should try to create a physics constraint between this object and the cursor. 
The only other place this tag is used is in choosing which action sets should be active on the next frame. If you need the `right/leftCursorHoveringOnInteractable` action set to be active, then you should give this object a new tag and add it to the check in `resolveActionSets`.